assets/js/components/wp-dashboard/WPDashboardWidgets.js

```diff
@@ -89,6 +90,7 @@ const WPDashboardWidgets = () => {
 				}
 			) }
 		>
+			<GoogleChart />
```

Comment:
This looks like it was added for testing but got left in 😄

Reply:
Argh, yes, whoops.

![shame-cube](https://user-images.githubusercontent.com/90871/211849539-22496e24-0a7c-41a1-971c-97fb528f71f4.gif)